README.md

```diff
@@ -27,14 +28,14 @@ Install from the Arch User Repository: `flashfocus`
 
 ```
 sudo apt-get install libxcb-render0-dev libffi-dev python-dev python-cffi
-pip install flashfocus
+pipx install flashfocus
```

Comment:
I like pipx and use it myself, but I think we should stick with pip here. Its the official package manager and anyone technical enough to be using pipx will know how to translate the pip command anyway

Reply:
I see no reason why to stick to pip. On debian & derivatives pip won't even install applications as per `PEP 668` outside virtual envs. This is the very usecase pipx is meant for.

Reply:
Sorry I'd rather leave it as is for now. Perhaps pipx will eventually become ubiquitous but I don't think we're quite there yet. I took a look at the install instructions for a few of the most popular python apps (django, flask and ansible) and all three still reference pip. I do think `pip install --user` would be an improvement though, so that the command at least works.

> On debian & derivatives pip won't even install applications as per PEP 668 outside virtual envs

This is a good point, but those users should probably be using the debian package instead.